streams/src/test/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionResponseWrapperSerdeTest.java

```diff
@@ -17,25 +17,68 @@
 package org.apache.kafka.streams.kstream.internals.foreignkeyjoin;
 
 import org.apache.kafka.common.errors.UnsupportedVersionException;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.streams.state.internals.Murmur3;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
 import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
 public class SubscriptionResponseWrapperSerdeTest {
+    private static final class NonNullableSerde<T> implements Serde<T>, Serializer<T>, Deserializer<T> {
```

Comment:
Adding this class (and using it below) causes NPEs in the current tests without the corresponding fix.

Reply:
Ah ok fair enough -- thanks!